mmcv/image/misc.py

```diff
@@ -9,15 +9,18 @@
     torch = None
 
 
-def tensor2imgs(tensor, mean=(0, 0, 0), std=(1, 1, 1), to_rgb=True):
+def tensor2imgs(tensor, mean=None, std=None, to_rgb=True):
     """Convert tensor to 3-channel images or 1-channel gray images.
 
     Args:
         tensor (torch.Tensor): Tensor that contains multiple images, shape (
             N, C, H, W). :math:`C` can be either 3 or 1.
-        mean (tuple[float], optional): Mean of images. Defaults to (0, 0, 0).
-        std (tuple[float], optional): Standard deviation of images.
-            Defaults to (1, 1, 1).
+        mean (tuple[float], optional): Mean of images. If None, default one
+            will be used according to the channel dim of tensor.
```

Comment:
"default one" is ambiguous. Suggested change: "`(0, )` or `(0, 0, 0)` will be used..."